### PR TITLE
Add confirmation support.

### PR DESCRIPTION
**Pull request by ricklamers**
Also added basic utility scripts for running both servers during development with a single command.

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+trap "exit" INT TERM ERR
+trap "kill 0" EXIT
+
+(
+  cd examples/fast-api-server
+  poetry run ./run.sh
```
Comment (colemccracken) [inline scripts/run.sh:8]: Took a comment to slack about this

**Review by colemccracken — APPROVED** [review]

File: scripts/run.sh (hunk 1 above)
Comment (ricklamers) [reply scripts/run.sh]: What about in the Makefile instead?

Comment (colemccracken) [reply scripts/run.sh]: I dont have strong opinions either way. Happy to ship and revisit later if it becomes a problem

**Review by jordan-definitive — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -12,3 +12,13 @@ publish-openassistants-fastapi:
...
 	@cd packages/openassistants-ui && npm publish
+
+refresh-poetry:
+	@echo "Refreshing Poetry environment..."
+	@cd examples/fast-api-server && poetry install
+
+run:
+	@echo "Running services in background..."
+	@(cd examples/fast-api-server && poetry run ./run.sh) &
+	@(cd examples/next && yarn dev) &
```
Comment (jordan-definitive) [inline Makefile:23]: break out as separate tasks?

Comment (ricklamers) [reply Makefile]: Good call

**Pull request merged**